provider/import.go

```diff
@@ -81,11 +81,11 @@ func Import(ctx context.Context, p Provider, hcl, tfstate writer.Writer, f *filt
 				if err != nil {
 					cause := errors.Cause(err)
 
-					// All those ignored errors are types of errors to identify that something happend, but not that something went wrong
-					// some of them are to skip the resource and others just informative
-					// we'll continue to the next resource
-					fmt.Fprintf(out, "\ncould not read resource: %s.%s\n", re.Type(), re.ID())
-					logger.Log("error", cause)
+					// Error are ignored. If a resource is invalid we assume it can be skipped.
+					// Log errors for verbose and debug mode.
+
+					logger.Log("msg", fmt.Sprintf("[ERR] : %s", cause))
```

Comment:
I would say that instead of prefixing, to use an specific KV, like instead of `msg` use `error`, and just set the cause.